freanalysis_radiation/freanalysis_radiation/__init__.py

```diff
@@ -15,9 +15,9 @@
 class Metadata:
     activity_id: str = "dev"
     institution_id: str = ""
-    source_id: str = ""
-    experiment_id: str = "c96L65_am5f4b4r1-newrad_amip"
-    frequency: str = "monthly"
+    source_id: str = "am5"
+    experiment_id: str = "c96L65_am5f7b11r0_amip"
+    frequency: str = "P1M"
```

Comment:
Same thing. Looks good, and I think we'll go to "mon" for frequency in the future to be standard.